Library/Homebrew/tap.rb

```diff
@@ -1352,7 +1352,12 @@ def core_cask_tap?
 
   sig { params(token: String).returns(Pathname) }
   def new_cask_path(token)
-    cask_subdir = token[0].to_s
+    cask_subdir = if token.start_with?("font-")
+      "fonts/#{token.delete_prefix("font-")}"
```

Comment:
```suggestion
      "fonts/#{token.delete_prefix("font-").first}"
```
Sorry, missed this from my first suggestion

Reply:
Nice. Thanks!